docs/docs/concepts/advanced/contract_creation.md

```diff
@@ -325,6 +325,6 @@ As an extension of this example, the deployer might wish bake-into the private c
 
 Adds initial public state variables to the public data tree.
 
-## Participate
```

Comment:
Do we not still want to signal this?

Reply:
in general, yes, but it is randomly on a few pages and is not clean. I think we should consolidate this info into 1 page. I'll clean up the other random pages that have this CTA

Reply:
Just added the "Participate" section to the landing page.